artifactory/utils/transfersettings.go

```diff
@@ -22,7 +22,8 @@ const (
 )
 
 type TransferSettings struct {
-	ThreadsNumber int `json:"threadsNumber,omitempty"`
+	ThreadsNumber int    `json:"threadsNumber,omitempty"`
+	LogLevel      string `json:"logLevel,omitempty"`
```

Comment:
I think it will be safer to work with the same log-level type, using our methods for handling the log levels.
```suggestion
	LogLevel      log.LevelType `json:"logLevel,omitempty"`
```